Week3/homework/5-step3.js

```diff
@@ -1,2 +1,7 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+for(var i=0; i < arr.length; i++){
+    for (var j=0; j < arr[i].length; j++){
+      product = product * arr[i][j];
+    }
+  }
```

Comment:
This code by itself is incomplete: it will not run. You copied only the code you wrote yourself from the freeCodeCamp exercise. The surrounding code, needed to make this run, is missing.

Note that you can use the short hand `*=` operator here:

```js
product *= arr[i][j];
```